src/gr11review/part1/Review4.java

```diff
@@ -1 +1,45 @@
 package gr11review.part1;
+import java.io.*;
+import java.text.*;
+
+/**
+* Take the amount of items the user wants to buy. Ask for the prices using a loop that runs the exact amount of times as the number of items the user is buying.
+* Display the subtotal, display the 13% tax, and display the total
+@author Benjamin Teh
+*/
+
+public class Review4{
+  public static void main(String[] args) throws IOException{
+    BufferedReader keyboard = new BufferedReader(new InputStreamReader(System.in));
+    NumberFormat numberFormat = new DecimalFormat("#,###.00");
+
+    int itemCount;
+    double itemPrice;
+    double subTotal = 0;
+    double salesTax;
+    double finalTotal;
+    int priceCounter;
+
+    System.out.println("How many items do you want to buy?");
```

Comment:
prompts should be `System.out.print` instead of `System.out.println` so that input can be on the same line as the prompt.